torch/utils/filelock.py

```diff
@@ -0,0 +1,37 @@
+from types import TracebackType
```

Comment:
The name of the module here is suspect. I don't think we actually want to be in the business of providing a public filelock utility, i.e., I don't think this should be covered in BC guarantees. So `torch.utils._filelock` would be better.

Reply:
Good catch, renamed.